.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file. I suggest you add `.vscode/` to your global `.gitignore` (i.e. `~/.gitignore`)

Reply:
OK, my bad.
I have removed it.

Reply:
It's ok. Thanks!